substrate/frame/multisig/src/migrations.rs

```diff
@@ -85,3 +88,119 @@ pub mod v1 {
 		}
 	}
 }
+
```

Comment:
We should use `VersionedMigration` here. @liamaharon can provide guidance if needed.